packages/itwin/grouping-mapping-widget/src/widget/components/groupsHelpers.ts

```diff
@@ -134,3 +155,115 @@ export const getHiliteIdsAndKeysetFromGroup = async (
   }
 
 };
+
+const processGroupIds = async (
```

Comment:
Function name is too generic. What does "process" mean in this instance?

Reply:
Renamed it to `getHiliteIdsForGroup`